docs/README.md

```diff
@@ -34,6 +34,8 @@ can fix it.
        * [Disabling Side-channel Mitigations](sidechannels.md)
        * [Parallel Simulations](parallel_sims.md)
      * [Migrating Simulations from Shadow 1.x](migrating_from_1x.md)
+     * Application and Library-Specific Notes
+       * [libopenblas](libopenblas.md)
```

Comment:
The indent spacing between the "Performance Tuning" subsection and the "Developer Guides" section appears inconsistent with the rest of the outline.

Reply:
I *think* it's a github rendering bug. It looks fine locally and here: https://raw.githubusercontent.com/shadow/shadow/aee287fac7ed58b96e3015c879b8a87f14595315/docs/README.md